doc/model/fluid.rst

```diff
@@ -0,0 +1,176 @@
+equations for a completely inviscid fluid. *MIRGE-Com* provides an Euler operator, with associated
+utilities functions, for solving flows of this type.  Inviscid fluxes and utilities are found in
+:mod:`mirgecom.inviscid`, and the Euler operator for the RHS in :mod:`mirgecom.euler`.
+
+.. _viscous-stress-tensor:
+
+Viscous stress tensor
+---------------------
+The viscous stress tensor has components:
+
+.. math::
+    \tau_{ij} = \mu \left(\partial_j{v_i} + \partial_i{v_j}\right)
+    +(\mu_B - \frac{2}{3}\mu)\partial_k{v_k}\delta_{ij}
+
+with fluid velocity components ${v}_{i}$, the first coefficient of fluid
+viscosity $\mu$, and bulk viscosity $\mu_B$.
+
+
+.. _diffusive-flux:
+
+Diffusive flux
+--------------
+The species diffusive fluxes are given by:
+
+.. math::
+   \mathbf{J}_{\alpha} = -\rho{d}_{(\alpha)}\nabla{Y}_{\alpha},
+
+with gas density $\rho$, species diffusivities ${d}_{\alpha}$, and
+species mass fractions ${Y}_{\alpha}$.  The parens $(\alpha)$ indicate no sum
+over repeated indices is to be performed.
+
+
+.. _heat-flux:
+
+Heat flux
+---------
+
+The total heat flux $\mathbf{q}$ is calculated as the sum of the
+conductive and diffusive components, $\mathbf{q}_{c}$ and $\mathbf{q}_{d}$,
+respectively:
+
+.. math::
+   \mathbf{q} = \mathbf{q}_c + \mathbf{q}_d
+
+
+Conductive heat flux
+^^^^^^^^^^^^^^^^^^^^
+The conductive heat flux vector is defined directly from Fourier's law of thermal conduction:
+
+.. math::
+    \mathbf{q}_c = -\kappa\nabla{T},
+
+where $\kappa$ is the thermal conductivity, and ${T}$ is the gas
+temperature.
+
+Diffusive heat flux
+^^^^^^^^^^^^^^^^^^^
+The diffusive heat flux vector is defined as
+
+.. math::
+   \mathbf{q}_d = {h}_{\alpha}\mathbf{J}_{\alpha},
+
+with the species specific enthalpy ${h}_{\alpha}$, and the species
+diffusive flux vector $\mathbf{J}_{\alpha}$.
+
+.. _Chemistry:
+
+Chemistry
+---------
+
+Chemical reactions introduce source terms in the energy and species conservation equations.
+The species source term is the amount of mass produced for each species:
+
+.. math::
+   W^{\mathtt{chem}}_{\alpha} = w_{(\alpha)}\dot{\omega}_{\alpha},
+
+where ${w}_{\alpha}$ is the molecular weight of each species, and $\dot{\omega}_{\alpha}$ is the net
+chemical production rate for each species. Here, the parens $(\alpha)$ indicates no sum is to be performed
+over repeated indices. 
+
+The energy source term is the amount of thermal energy used to create each species:
+
+.. math::
+   E^{\mathtt{chem}} = -h^f_{\alpha}W^{\mathtt{chem}}_{\alpha},
+
+where $h^f_{\alpha}$ is the enthalpy of formation for each species.
+
+.. _eos-and-matprop:
+
+Equations of State and Material properties
+------------------------------------------
+
+Equations of state (EOS) provide functions that relate the fluid state $Q$, and the
+thermodynamic properties such as pressure $p$, temperature $T$, specific enthalpies $h_{\alpha}$,
```

Comment:
It would help me here to know who is a function of what in the most general case.

Reply:
Ok, let's talk about this.  We can write one such relation that is relevant for say mixtures, in the form (p,T,h) = F(Q), but the EOS actually uses inversions and other relations depending on the situation and/or known quantities. (See e.g. https://en.wikipedia.org/wiki/Maxwell_relations, for some examples).  Anything we write here will just be one example of a form of *F(Q)* (or similar).

Reply:
See how this grabs ya. (a8cd8e1)